scripts/update/01-run.sh

```diff
@@ -113,6 +113,40 @@ EOF
 cd "$UMBREL_ROOT"
 ./scripts/stop
 
+# Move Docker data dir to external storage now if this is an old install.
+# This is only needed temporarily until all users have transitioned Docker to SSD.
+DOCKER_DIR="/var/lib/docker"
+MOUNT_POINT="/mnt/data"
+EXTERNAL_DOCKER_DIR="${MOUNT_POINT}/docker"
+if [[ ! -z "${UMBREL_OS:-}" ]] && [[ ! -d "${EXTERNAL_DOCKER_DIR}" ]]; then
+  echo "Attempting to move Docker to external storage..."
+
+  echo "Stopping Docker service..."
+  service docker stop
```

Comment:
```suggestion
  systemctl stop docker
```
Not really important, but I'd use `systemctl`, while service is compatible, according to the man page, it was made for sysV, so I wouldn't use it.
Same below.

Reply:
Good spot!

Resolved in ca8a988